### DISCUSSION TITLE
C++ Project for Cassandra

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/PyCI#3] [General]
@msricher as part of @cassmasschelein 's  CSE701 course, she needs to write some C++ code. It can be on anything, so perhaps there are bits of PyCI where some coding can be helpful? 

There may be other repos too; let me know what you think.

**Comment by PaulWAyers** [comment]
@msricher @cassmasschelein I think we have @cassmasschelein 's permanent code in decent shape now, yes? Can we exploit it for APIG for this release?

**Reply by msricher** [reply]
Not yet.  Once she gets back to me we will finish it.
